platform/src/config.rs

```diff
@@ -0,0 +1,24 @@
+use roc_std::{ RocList, RocStr };
+
+#[derive(Default)]
+#[repr(C)]
+pub struct Config {
+    pub outputFilePath: RocStr,
+    pub subtitle: RocStr,
+    pub title: RocStr,
+    pub height: u32,
+    pub width: u32,
+    pub x: RocList<i32>,
+    pub y1: RocList<i32>,
```

Comment:
I assume y would be a list of lists long term?

Reply:
Yes - I made them all shallow lists in the interest of incremental debugging, but I'd love to replace all 3 of these lists with one `values` or `data` structure.